### PR TITLE
Feat/styling

### LINKED ISSUE #4: Setup Responsiveness

### LINKED ISSUE #23: Responsive Design

### PR DESCRIPTION
**Pull request by tenthwalker**
What I did:
This pull request adds styling to bring the project in line with the wireframe and style inspiration. It updates colors, borders, layouts, class/id naming where necessary, responsive design flexibility, and removes placeholder/boilerplate code.

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Incorporation of Auth0 to nail down how that will impact any further styling.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```
Comment (Jesuitman) [inline public/index.html:13]: good job cleaning this up! i didnt even thin kto check that out

File: src/App/App.css [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
   justify-content: space-between;
   align-items: center;
   padding: 20px;
-  background-color: #f0f0f0;
 }
 
 .header-text {
-  color: blue;
+  background-color: #FDFFFC;
+  border-bottom: .5rem solid #235789;
```
Comment (Jesuitman) [inline src/App/App.css:20]: love the responsive design!

File: src/Card/Card.css [hunk 3]
```diff
@@ -1,25 +1,38 @@
...
+
+.card h2 {
+  font-size: 18px;
+  margin-bottom: 5px;
+}
+
+.card p {
+  font-size: 14px;
+  flex-grow: 1;
+  overflow: hidden;
```
Comment (Jesuitman) [inline src/Card/Card.css:29]: good use of overflow hidden!

File: src/Card/Card.js [hunk 4]
```diff
@@ -16,11 +16,11 @@ const Card = ({ snippet }) => {
 
   return (
     <div className="card">
-      <div className="card-content" onClick={handleTitleClick}>
+      <article className="card-content" onClick={handleTitleClick}>
```
Comment (Jesuitman) [inline src/Card/Card.js:19]: awesome job limiting our divs!

File: src/Search/Search.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+#resultName {
+  align-self: center;
+  color: #161925;
+  background-color: #FDFFFC;
+  border-bottom: 0.5rem solid #C1292E;
+  border-right: 0.5rem solid #C1292E;
+  width: fit-content;
+}
```
Comment (Jesuitman) [inline src/Search/Search.css:37]: great primary color usage!

**Review by JoeyStewart — APPROVED** [review]

**Comment by JoeyStewart** [comment]
Incredible work!

File: src/Search/Search.js [hunk 6]
```diff
@@ -32,22 +33,22 @@ function WikipediaSearch() {
...
-      .then((response) => {
-        if (!response.ok) {
-          throw new Error('Network response was not ok');
-        }
-        return response.json();
-      })
-      .then((data) => {
+  await fetch(url + new URLSearchParams(params).toString())
+    .then((response) => {
+      if (!response.ok) {
```
Comment (Jesuitman) [inline src/Search/Search.js:38]: that spacing looks incredible, i love that you cleaned so much up

File: src/Search/Search.css [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1 200px;
+  justify-content: space-evenly;
+}
+
+#resultName {
+  align-self: center;
+  color: #161925;
+  background-color: #FDFFFC;
```
Comment (MaryBruff) [inline src/Search/Search.css:33]: Yeah the primary colors turned out really well! I love how it looks!

File: src/WikipediaPage/WikipediaPage.css [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
-    background-color: black;
-    color: white;
-    padding: 10px 20px;
-    border: none;
-    border-radius: 5px;
-    cursor: pointer;
-    transition: background-color 0.3s ease;
-  }
+.saveButton, .favoriteButton {
+  background-color: #161925;
```
Comment (Jesuitman) [inline src/WikipediaPage/WikipediaPage.css:8]: the use of primary colors makes it really pop off, well done

**Review by Jesuitman — APPROVED** [review]

File: src/Card/Card.js (hunk 4 above)
Comment (nicktu12) [reply src/Card/Card.js:19]: love the comments happening in here!

File: src/Search/Search.css [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  form {
+    height: fit-content;
+    align-self: center;
+  }
+  .search-banner {
+    justify-content: space-evenly
+  }
+}
+
+@media screen and (max-width: 500px) {
```
Comment (nicktu12) [inline src/Search/Search.css:49]: media queries 💪🏻

File: src/Search/Search.js [hunk 10]
```diff
@@ -32,22 +33,22 @@ function WikipediaSearch() {
...
+    .then((data) => {
 // Sets the state of the initial results to the first result of the search (i.e. Rush Limbaugh)
-        setInitialResults(data.query.search[0]);
-        navigate(`/article/${data.query.search[0].pageid}`);
-      })
-      .catch((error) => {
-        console.error('There was a problem with the fetch operation:', error);
-      });
-  }
+      setInitialResults(data.query.search[0]);
```
Comment (nicktu12) [inline src/Search/Search.js:45]: `data.query.search[0]` is quite a handful - could you store this in a variable with an appropriate variable name to improve readability?

**Pull request merged**